_sass/content.scss

```diff
@@ -10,10 +10,11 @@
       margin-top: 1em;
     }
   }
+  background-color: blue;
 }
 
 .content a:link, .content a:visited {
-  color: blue;
```

Comment:
The blue makes the text hard to read. I would choose a lighter color here.

Reply:
I respectfully disagree. Most people's retinas inherently receive blue very well

Reply:
I actually really like this blue.  @kaylaewood, maybe you should get your eyes checked.  Let me know if you need a good ophthalmologist.